[dynamo] Avoid divided by zero error when printing out choices

yinghai opened the pull request:
Summary: We met with problem in practice. Although I think whenever we meet with this problem, something bad probably happened in upstream. Like the run instance possibly returned immediately due to error. Throw this out to see if we can catch something earlier. That'll be better.

Test Plan: CI




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov

ezyang reviewed (approved):
fine as bandaid. check phab comments

huydhn commented:
@pytorchbot rebase

yinghai commented:
@pytorchbot merge -i

huydhn commented:
@pytorchbot merge -i

huydhn commented:
@pytorchbot merge

huydhn commented:
@pytorchbot merge

Commit:
[dynamo] Avoid divided by zero error when printing out choices (#109328)

Summary:

We met with problem in practice. Although I think whenever we meet with this problem, something bad probably happened in upstream. Like the run instance possibly returned immediately due to error. Throw this out to see if we can catch something earlier. That'll be better.

bypass-github-pytorch-ci-checks

Test Plan: CI

Reviewed By: ezyang, huydhn

Differential Revision: D49295671
